flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -54,6 +54,7 @@ run_test "Wordcount end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wo
 run_test "Shaded Hadoop S3A end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh hadoop"
 run_test "Shaded Presto S3 end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh presto"
 run_test "Custom FS plugin end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh dummy-fs"
+run_test "Custom FS plugin isolation end-to-end test" "$END_TO_END_DIR/test-scripts/test_plugins_isolation.sh"
```

Comment:
I don't think we should be adding a separate test for the plugin isolation. It will make the tests unnecessarily execute longer. This new test should supersede the previous `test_bach_wordcount.sh`. 

In other words, `test_batch_wordcount.sh` could be expanded to support multiple (or at least 1 or 2) input files (the expected result would be different for different amount of parameters) and all of the previous usages of the `dummy-fs` should be now using both `dummy-fs` and `another-dummy-fs` file systems.